internal/sponge/sponge.go

```diff
@@ -0,0 +1,120 @@
+// Package sponge provides support for item importing.
+package sponge
+
+import (
+	"reflect"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+	"github.com/cayleygraph/cayley"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	"github.com/coralproject/shelf/internal/wire"
+)
+
+// Import imports an item into the items collections and into the graph database.
+func Import(context interface{}, db *db.DB, graph *cayley.Handle, itm *item.Item) error {
+	log.Dev(context, "Import", "Started : ID[%s]", itm.ID)
+
+	// See if the item already exists.
+	if itm.ID != "" {
```

Comment:
Change the comment to say.

If the item already exists then ....  and describe why we have this extended code inside the if.